client/websocket_client.tsx

```diff
@@ -1,10 +1,14 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {SocketEvents} from 'utils/constants';
+
 const MAX_WEBSOCKET_FAILS = 7;
 const MIN_WEBSOCKET_RETRY_TIME = 3000; // 3 sec
 const MAX_WEBSOCKET_RETRY_TIME = 300000; // 5 mins
 
+const reliableWebSockets = true;
```

Comment:
This needs to be changed to actually take from the server config `ServiceSettings.EnableReliableWebSockets`